centreon/packages/ui/src/Dialog/index.tsx

```diff
@@ -28,12 +28,18 @@ const useStyles = makeStyles<StylesProps>()((theme, { contentWidth }) => ({
   }
 }));
 
+interface DialogDataTestId {
+  dataTestIdCanceledButton?: string;
+  dataTestIdConfirmButton?: string;
+}
+
 export type Props = {
   cancelDisabled?: boolean;
   children?: ReactNode;
   className?: string;
   confirmDisabled?: boolean;
   contentWidth?: number;
+  dataTestId?: DialogDataTestId;
```

Comment:
fyi. there are generic type props available for data-testId and aria-label to extend from

```
import { AriaLabelingAttributes } from '../@types/aria-attributes';
import { DataTestAttributes } from '../@types/data-attributes';

type Props = {
...
} & AriaLabelingAttributes &
  DataTestAttributes;
```

this way we can avoid a mix of `data-testid` and `dataTestId` on the components

Reply:
and can be used as

```
const Component = ({
  children,
  ...attr
}: Props): ReactElement => {

  return (
    <div
      {...attr}
    >
      {children}
    </div>
  );
};
```

Reply:
in my case the dialog contains 2 buttons , I should pass 2 data-testId , the type from @types/data-attributes it s generic , I could pass just one data-testid